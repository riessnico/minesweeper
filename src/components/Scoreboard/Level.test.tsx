import React from 'react';
import { render } from '@testing-library/react';

import { Level } from './Level';

it('Level renders correctly', () => {
  const { asFragment } = render(
    <Level>{['beginner', 'intermediate', 'expert']}</Level>,
  );

  expect(asFragment()).toMatchSnapshot();
});
