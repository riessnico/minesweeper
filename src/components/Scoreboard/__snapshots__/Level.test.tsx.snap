// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Level renders correctly 1`] = `
<DocumentFragment>
  <select
    class="css-160ys9i"
  >
    <option
      class="css-zcmdxr"
      value="beginner"
    >
      beginner
    </option>
    <option
      class="css-zcmdxr"
      value="intermediate"
    >
      intermediate
    </option>
    <option
      class="css-zcmdxr"
      value="expert"
    >
      expert
    </option>
  </select>
</DocumentFragment>
`;
